src/java/picard/sam/BamIndexStats.java

```diff
@@ -45,9 +45,19 @@
  * @author Martha Borkan
  */
 @CommandLineProgramProperties(
-        usage = "Generates BAM index statistics, including the number of aligned and unaligned SAMRecords for each reference sequence, " +
-                "and the number of SAMRecords with no coordinate." +
-                "Input BAM file must have a corresponding index file.\n",
+        usage = "Command line program to print statistics from BAM index (.bai) file.  Statistics include count of " +

```

Comment:
No need to specify that it's a command line program -- they're all command line programs.